products/angular.md

```diff
@@ -97,4 +106,8 @@ months, with 1-3 minor releases for every major release. There is an
 [update guide](https://angular.io/guide/updating "Keeping your Angular projects up-to-date")
 available.
 
+Commercial support is available for all deprecated versions of Angular through the 
+[HeroDevs Never-Ending Support](https://www.herodevs.com/support/nes-angular) initiative. 
+For more information about commercial support options visit [HeroDevs' Website](https://www.herodevs.com/). 
```

Comment:
The last sentence sounds like an advertisement and does not provide more details as in the first, I suggest to remove:
```suggestion
Commercial support is available for all deprecated versions of Angular through the 
[HeroDevs Never-Ending Support](https://www.herodevs.com/support/nes-angular) initiative. 
```